frame/assets/src/lib.rs

```diff
@@ -473,25 +483,37 @@ pub mod pallet {
 				Ok(_) => None,
 				Err(origin) => Some(ensure_signed(origin)?),
 			};
-			Asset::<T>::try_mutate_exists(id, |maybe_details| {
-				let mut details = maybe_details.take().ok_or(Error::<T>::Unknown)?;
+			Asset::<T, I>::try_mutate_exists(id, |maybe_details| {
+				let mut details = maybe_details.take().ok_or(Error::<T, I>::Unknown)?;
 				if let Some(check_owner) = maybe_check_owner {
-					ensure!(details.owner == check_owner, Error::<T>::NoPermission);
+					ensure!(details.owner == check_owner, Error::<T, I>::NoPermission);
 				}
-				ensure!(details.accounts == witness.accounts, Error::<T>::BadWitness);
-				ensure!(details.sufficients == witness.sufficients, Error::<T>::BadWitness);
-				ensure!(details.approvals == witness.approvals, Error::<T>::BadWitness);
-
-				for (who, v) in Account::<T>::drain_prefix(id) {
+				ensure!(
+					details.accounts == witness.accounts,
+					Error::<T, I>::BadWitness
+				);
```

Comment:
in substrate the line length is 100 character, those ensure doesn't need to go to newline.